pkg/devfile/adapters/common/generic.go

```diff
@@ -64,7 +64,11 @@ func (a GenericAdapter) ExecuteCommand(compInfo ComponentInfo, command []string,
 
 // ExecuteDevfileCommand executes the devfile init, build and test command actions synchronously
 func (a GenericAdapter) ExecuteDevfileCommand(command common.DevfileCommand, show bool) error {
-	c, err := New(command, a.Devfile.Data.GetCommands(), a)
+	commands, err := a.Devfile.Data.GetCommands()
```

Comment:
can we rename the var to `commandsMap`/`devfileCommandsMap`? This file's command names are all over the place

Reply:
Maybe `commandsMap` since all other files are calling it that